local_intelligence_hub/settings.py

```diff
@@ -52,9 +55,14 @@
 MAPIT_API_KEY = env("MAPIT_API_KEY")
 GOOGLE_ANALYTICS = env("GOOGLE_ANALYTICS")
 GOOGLE_SITE_VERIFICATION = env("GOOGLE_SITE_VERIFICATION")
-TEST_AIRTABLE_BASE_ID = env("TEST_AIRTABLE_BASE_ID")
-TEST_AIRTABLE_TABLE_NAME = env("TEST_AIRTABLE_TABLE_NAME")
-TEST_AIRTABLE_API_KEY = env("TEST_AIRTABLE_API_KEY")
+TEST_AIRTABLE_MEMBERLIST_BASE_ID = env("TEST_AIRTABLE_MEMBERLIST_BASE_ID")
+TEST_AIRTABLE_MEMBERLIST_TABLE_NAME = env("TEST_AIRTABLE_MEMBERLIST_TABLE_NAME")
+TEST_AIRTABLE_MEMBERLIST_API_KEY = env("TEST_AIRTABLE_MEMBERLIST_API_KEY")
```

Comment:
Minor suggestion. These should live in the test environment settings, not in the default settings for all environments.

Reply:
Going to park this as a ticket for someone to figure out. https://linear.app/commonknowledge/issue/MEEP-211/separate-settingspy-file-for-dev-testing-and-production